modules/installation-extend-edge-nodes-aws-local-zones.adoc

```diff
@@ -0,0 +1,120 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/cluster-tasks.adoc
+
+ifeval::["{context}" == "installing-aws-localzone"]
+:localzone:
+endif::[]
+
+:_content-type: PROCEDURE
+[id="installation-extend-edge-nodes-aws-local-zones_{context}"]
+= Creating user workloads in AWS Local Zones
+After you create an Amazon Web Service (AWS) Local Zone environment, and you deploy your cluster, you can use edge worker nodes to create user workloads in Local Zone subnets. 
+
+After the `openshift-installer` creates the cluster, the installation program automatically specifies a taint effect of `NoSchedule` to each edge worker node. This means that a scheduler does not add a new pod, or deployment, to a node if the pod does not match the specified tolerations for a taint. You can modify the taint for better control over how each node creates a workload in each Local Zone subnet.
+
+The `openshift-installer` creates the machine set manifests file with `node-role.kubernetes.io/edge` and `node-role.kubernetes.io/worker` labels applied to each edge worker node that is located in a Local Zone subnet.
+
+.Prerequisites
+
+* You have an access to the OpenShift CLI (`oc`).
+* You deployed your cluster in a Virtual Private Cloud (VPC) with defined Local Zone subnets.
+* You ensured that the machine set for the edge workers on Local Zone subnets specifies the taints for `node-role.kubernetes.io/edge`.
+
+.Procedure
+
+. Create a `deployment` resource YAML file for an example application to be deployed in the edge worker node that operates in a Local Zone subnet. Ensure that you specify the correct tolerations that match the taints for the edge worker node.
++
+.Example of a configured `deployment` resource for an edge worker node that operates in a Local Zone subnet
+[source,yaml]
+----
+kind: Namespace
+apiVersion: v1
+metadata:
+  name: <local_zone_application_name>
+---
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: <pvc_name>
+  namespace: <local_zone_application_namespace>
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 10Gi
+  storageClassName: gp2-csi <1>
+  volumeMode: Filesystem
+---
+apiVersion: apps/v1
+kind: Deployment <2>
+metadata:
+  name: <local_zone_application> <3>
+  namespace: <local_zone_application_namespace> <4>
+spec:
+  selector: 
+    matchLabels:
+      app: <local_zone_application>
+  replicas: 1
+  template:
+    metadata:
+      labels: 
+        app: <local_zone_application>
+        zone-group: ${ZONE_GROUP_NAME} <5>
+    spec:
```

Comment:
@dfitzmau you need to add those lines defining the security context to prevent the warning mentioned [here](https://github.com/openshift/openshift-docs/pull/57063/files#r1161553904)

```suggestion
    spec:
      securityContext:
        seccompProfile:
          type: RuntimeDefault
```

@yunjiang29 would you mind validating?

```
cat <<EOF | oc create -f -
kind: Namespace
apiVersion: v1
metadata:
  name: lz-app-ns
---
kind: PersistentVolumeClaim
apiVersion: v1
metadata:
  name: lz-pvc
  namespace: lz-app-ns
spec:
  accessModes:
    - ReadWriteOnce
  resources:
    requests:
      storage: 10Gi
  storageClassName: gp2-csi 
  volumeMode: Filesystem
---
apiVersion: apps/v1
kind: Deployment 
metadata:
  name: lz-app
  namespace: lz-app-ns 
spec:
  selector:
    matchLabels:
      app: lz-app
  replicas: 1
  template:
    metadata:
      labels:
        app: lz-app
        machine.openshift.io/zone-group: ${ZONE_GROUP_NAME} 
    spec:
      securityContext:
        seccompProfile:
          type: RuntimeDefault
      nodeSelector: 
        machine.openshift.io/zone-group: ${ZONE_GROUP_NAME}
      tolerations: 
      - key: "node-role.kubernetes.io/edge"
        operator: "Equal"
        value: ""
        effect: "NoSchedule"
      containers:
        - image: openshift/origin-node
          command:
           - "/bin/socat"
          args:
            - TCP4-LISTEN:8080,reuseaddr,fork
            - EXEC:'/bin/bash -c \"printf \\\"HTTP/1.0 200 OK\r\n\r\n\\\"; sed -e \\\"/^\r/q\\\"\"'
          imagePullPolicy: Always
          name: echoserver
          ports:
            - containerPort: 8080
          volumeMounts:
            - mountPath: "/mnt/storage"
              name: data
      volumes:
      - name: data
        persistentVolumeClaim:
          claimName: lz-pvc
EOF
```

results

```
namespace/lz-app-ns created
persistentvolumeclaim/lz-pvc created
deployment.apps/lz-app created

```
I

Reply:
Thanks, @mtulio .

I updated the YAML file based on your suggestive addition.